README.md

```diff
@@ -1,5 +1,59 @@
 # Setup GitHub Action
 
+### Deprecation notice
+
+**Warning** This action has been deprecated.
+
+Please use the [pulumi/actions](https://github.com/pulumi/actions) action
+in [installation only mode](https://github.com/pulumi/actions#installation-only).
+</td></tr></table>
+
+#### Migrating to pulumi/actons
+
+To migrate, switch to `pulumi/actions@v4`.
```

Comment:
Should we be more specific and say `v4.1` because it is the minimum version containing install-only mode?

Reply:
Honestly, I was confused about the release process and how tagging for actions worked.
I always assumed the `@v{whatever}` was a constraint and GitHub does some kind of version resolution, but then I saw that the repository had a `v4` tag which suggested that maybe there's no version resolution.

Turns out that the release process moves the v4 tag to the latest v4 release.
So the v4 tag now points to 4.1.0
![image](https://user-images.githubusercontent.com/41730/224038859-9e7360db-95d0-4732-96cc-fc4c301a68a7.png)

So if we put v4.1.0 here, they'll be pinned exclusively to that version and never get something newer.